### PR TITLE
Improve German Translation

### PR DESCRIPTION
**Pull request by ChromexUnderscore**
+ Improved German Translations.
  + Corrected spelling mistakes with Translation for ``Gilded Netherite Armor``.
  + Replaced ``Patina`` with the german term``Edelrost``.
  + Replaced ``Hitman`` with ``Sniper``, because this term is better known in the german language.

### CODE REVIEW

File: src/main/resources/assets/additionaladditions/lang/de_de.json [hunk 1]
```diff
@@ -17,15 +17,15 @@
...
-  "item.additionaladditions.gilded_netherite_sword": "Vergoldeter Netheritschwert",
-  "item.additionaladditions.gilded_netherite_axe": "Vergoldeter Netheritaxt",
-  "item.additionaladditions.gilded_netherite_pickaxe": "Vergoldeter Netheritspitzhacke",
-  "item.additionaladditions.gilded_netherite_shovel": "Vergoldeter Netheritschaufel",
-  "item.additionaladditions.gilded_netherite_hoe": "Vergoldeter Netherithacke",
+  "item.additionaladditions.gilded_netherite_sword": "Vergoldetes Netheritschwert",
+  "item.additionaladditions.gilded_netherite_axe": "Vergoldete Netheritaxt",
+  "item.additionaladditions.gilded_netherite_pickaxe": "Vergoldete Netheritspitzhacke",
+  "item.additionaladditions.gilded_netherite_shovel": "Vergoldete Netheritschaufel",
+  "item.additionaladditions.gilded_netherite_hoe": "Vergoldete Netherithacke",
```
Comment (ChromexUnderscore) [inline src/main/resources/assets/additionaladditions/lang/de_de.json:24]: Wrong grammatical gender

File: src/main/resources/assets/additionaladditions/lang/de_de.json [hunk 2]
```diff
@@ -37,10 +37,10 @@
   "item.additionaladditions.chicken_nugget": "Chicken Nugget",
   "item.additionaladditions.gold_ring": "Goldring",
   "block.additionaladditions.glow_stick": "Leuchtstab",
-  "block.additionaladditions.copper_patina": "Kupfer Patina",
+  "block.additionaladditions.copper_patina": "Kupfer Edelrost",
```
Comment (ChromexUnderscore) [inline src/main/resources/assets/additionaladditions/lang/de_de.json:40]: ``Edelrost`` is the german term for ``Patina``.

File: src/main/resources/assets/additionaladditions/lang/de_de.json [hunk 3]
```diff
@@ -76,7 +76,7 @@
   "additionaladditions.advancements.obtain_pocket_jukebox.desc": "Stelle eine Taschen-Plattenspieler her, um unterwegs Musik zu hören",
   "additionaladditions.advancements.obtain_rose_gold": "Vergoldete Rosen?",
   "additionaladditions.advancements.obtain_rose_gold.desc": "Erhalte die Roségold-Rüstung",
-  "additionaladditions.advancements.obtain_scoped_crossbow": "Hitman",
+  "additionaladditions.advancements.obtain_scoped_crossbow": "Sniper",
```
Comment (ChromexUnderscore) [inline src/main/resources/assets/additionaladditions/lang/de_de.json:79]: ``Sniper`` is a more common term than ``Hitman`` in the german language.

File: src/main/resources/assets/additionaladditions/lang/de_de.json [hunk 4]
```diff
@@ -17,15 +17,15 @@
...
+  "item.additionaladditions.gilded_netherite_sword": "Vergoldetes Netheritschwert",
+  "item.additionaladditions.gilded_netherite_axe": "Vergoldete Netheritaxt",
+  "item.additionaladditions.gilded_netherite_pickaxe": "Vergoldete Netheritspitzhacke",
+  "item.additionaladditions.gilded_netherite_shovel": "Vergoldete Netheritschaufel",
+  "item.additionaladditions.gilded_netherite_hoe": "Vergoldete Netherithacke",
   "item.additionaladditions.gilded_netherite_helmet": "Vergoldeter Netherithelm",
   "item.additionaladditions.gilded_netherite_chestplate": "Vergoldeter Netheritharnisch",
   "item.additionaladditions.gilded_netherite_leggings": "Vergoldeter Netheritbeinschutz",
-  "item.additionaladditions.gilded_netherite_boots": "Vergoldeter Netheritstiefel",
+  "item.additionaladditions.gilded_netherite_boots": "Vergoldete Netheritstiefel",
```
Comment (ChromexUnderscore) [inline src/main/resources/assets/additionaladditions/lang/de_de.json:28]: Wrong grammatical gender

File: src/main/resources/assets/additionaladditions/lang/de_de.json [hunk 5]
```diff
@@ -37,10 +37,10 @@
   "item.additionaladditions.chicken_nugget": "Chicken Nugget",
   "item.additionaladditions.gold_ring": "Goldring",
   "block.additionaladditions.glow_stick": "Leuchtstab",
-  "block.additionaladditions.copper_patina": "Kupfer Patina",
+  "block.additionaladditions.copper_patina": "Kupfer Edelrost",
   "block.additionaladditions.rope": "Seil",
   "block.additionaladditions.amethyst_lamp": "Amethyst-Lampe",
-  "block.additionaladditions.patina_block": "Patinablock",
+  "block.additionaladditions.patina_block": "Edelrostblock",
```
Comment (ChromexUnderscore) [inline src/main/resources/assets/additionaladditions/lang/de_de.json:43]: ``Edelrost`` is the german term for ``Patina``.

**Comment by Dqu1J** [comment]
Thank you! I'm going to confirm that with the person who made the translations originally, and then merge if it's all good!

**Pull request merged**